Anature/src/application/controllers/overworld_cells/AbstractController.java

```diff
@@ -572,4 +572,17 @@ else if(!keys.isUp() && !keys.isDown() && !keys.isRight() && !keys.isLeft())
 			}
 		}
 	}
+
+	public void saveLoadUpdates()
+	{
+		KeysPressed keys = mView.getKeysPressed();
+		keys.turnOffAll();
+		keys.setCanMove(true);
+		
+		updatePcSprite();
+		
+		mView.hideDialogue();
+		mView.hideShopMenu();
+		mView.hideSideMenu();
```

Comment:
You need to add the line:
mCanTalkToTrainer = true;
to the saveAndLoad method in AbstractController. Without it, whenever you load a save, you will be unable to talk to any trainers.